packages/tools/src/tools/annotation/splines/renderer/canvas/CatmullRomRenderer.ts

```diff
@@ -0,0 +1,6 @@
+import { SplineCanvasRenderer } from './SplineCanvasRenderer';
```

Comment:
Not sure what are these files in the renderer/canvas and renderer/svg

Reply:
They shouldn't have been pushed